staging/src/k8s.io/apiserver/pkg/storage/etcd/metrics/metrics.go

```diff
@@ -59,6 +59,13 @@ var (
 		},
 		[]string{"operation", "type"},
 	)
+	objectCounts = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "apiserver_object_counts",
+			Help: "Current number of objects stored. Counts only types that have watch-cache enabled.",
```

Comment:
I don't think the comment is accurate... looks like it would work without the watch cache, right?

Reply:
Right.